gtk/src/toga_gtk/widgets/textinput.py

```diff
@@ -1,17 +1,35 @@
 from travertino.size import at_least
 
+from toga_gtk.keys import toga_key
+
 from ..libs import Gtk, gtk_alignment
 from .base import Widget
 
 
+class TogaConfirmListener:
```

Comment:
So - while we're only using this to catch the "confirm" event, it's a generic listener for keypress events. It would be preferable to keep this naming (and the onConfirm method) close to the underlying Android naming of the events they're catching, rather than the specific purpose that we care about those events.

Reply:
Actually - looking at this more - why use a class at all? GTK just requires a method. The old "gtk_key_release_event" was exactly what we needed. The extra class doesn't add anything but complexity.